packages/e2e/cypress/e2e/api/sqlRunner.cy.ts

```diff
@@ -1,285 +1,406 @@
-//                                         'credit_card',
-//                                     );
-//
-//                                     expect(results[1].payment_id).to.be.eq(2);
-//                                     expect(results[1].payment_method).to.be.eq(
-//                                         'credit_card',
-//                                     );
-//                                 });
-//                             } // Else keep polling
-//                             else if (retries < maxRetries) {
-//                                 poll(retries + 1);
-//                             } else {
-//                                 expect(
-//                                     resp.body.results.status,
-//                                     'Reached max number of retries without getting completed job',
-//                                 ).to.eq('completed');
-//                             }
-//                         });
-//                     };
-//                     poll();
-//                 });
-//             });
-//         });
-//     },
-// );
+// Object.entries({ postgres: warehouseConnections.postgresSQL }).forEach(  // For testing
+Object.entries(warehouseConnections).forEach(
+    // Object.entries({ snowflake: warehouseConnections.snowflake }).forEach(
+    // For testing
+    ([warehouseName, warehouseConfig]) => {
+        const getDatabaseDetails = () => {
+            switch (warehouseConfig.type) {
+                case WarehouseTypes.SNOWFLAKE:
+                    return [warehouseConfig.database, warehouseConfig.schema];
+                case WarehouseTypes.BIGQUERY:
+                    return [warehouseConfig.project, warehouseConfig.dataset];
+                case WarehouseTypes.REDSHIFT:
+                case WarehouseTypes.POSTGRES:
+                    return [warehouseConfig.dbname, warehouseConfig.schema];
+                default:
+                    return ['unknown', 'unknown'];
+            }
+        };
+        describe(`Get tables and fields for SQL runner on ${warehouseName} `, () => {
+            const projectName = `SqlRunner ${warehouseName} ${Date.now()}`;
+            let projectUuid: string;
+            beforeEach(() => {
+                cy.login();
+            });
+            before('create upstream project', () => {
+                cy.login();
+                cy.createProject(projectName, warehouseConfig).then((puuid) => {
+                    projectUuid = puuid;
+                });
+            });
+
+            after('delete upstream project', () => {
+                cy.log(`Deleting project by name ${projectName}`);
+                cy.deleteProjectsByName([projectName]);
+            });
+
+            // test for invalid sql
+            // note each db type may have different error messages
+            it.skip(`Get error on invalid SQL for ${warehouseName}`, () => {
```

Comment:
should we `unskip` this?

Reply:
that is currently failing, I don't have to fix that too, I'd rather merge it without this and fix it on another PR